template/$PROJECT_NAME$/lib/$PROJECT_NAME$/account/user.ex

```diff
@@ -23,7 +23,7 @@ defmodule <%= @project_name_camel_case %>.Account.User do
   end
 
   @spec hash_password(%Ecto.Changeset{}) :: %Ecto.Changeset{}
-  defp hash_password(changeset = %Ecto.Changeset{valid?: true, changes: %{password: pass}}) do
+  defp hash_password(%Ecto.Changeset{valid?: true, changes: %{password: pass}} = changeset) do
```

Comment:
Ah interesting. Well as I don't have an opinion and it doesn't change anything functionally, This looks good!